File: VB/RemoveNewButtonInLookups.Module.Win/WinDeactivateNewActionInLookupsController.vb
Imports DevExpress.ExpressApp.Editors

Namespace RemoveNewButtonInLookups.Module.Win
    Public Class WinDeactivateNewActionInLookupsController
        Inherits DeactivateNewActionInLookupsController

        Protected Overrides Function IsLookupTemplate() As Boolean
            Return TypeOf Frame.Template Is ILookupPopupFrameTemplate
        End Function
    End Class
End Namespace

File: CS/RemoveNewButtonInLookups.Module.Win/WinDeactivateNewActionInLookupsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.ExpressApp.Editors;

namespace RemoveNewButtonInLookups.Module.Win {
    public class WinDeactivateNewActionInLookupsController : DeactivateNewActionInLookupsController {
        protected override bool IsLookupTemplate() {
            return Frame.Template is ILookupPopupFrameTemplate;
        }
    }
}
